util/util.go

```diff
@@ -64,6 +67,8 @@ func RandomString(n int) string {
 
 // ModifyImageTag takes an imageName (e.g., registry/repo:tag), and returns an image name with updated tag
 func ModifyImageTag(imageName, tagName string) (string, error) {
+	normalisedTagName := KubernetesVersionToImageTag(NormalizedBuildVersion(tagName))
```

Comment:
This changes the behavior of the exported `util.ModifyImageTag` which unfortunately makes our API incompatible with the previous version.

We can possibly make another function call this one instead of changing it